src/main/java/com/shanebeestudios/skbee/elements/other/expressions/ExprNamespaceKeyOf.java

```diff
@@ -0,0 +1,52 @@
+package com.shanebeestudios.skbee.elements.other.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.NamespacedKey;
+import org.jetbrains.annotations.Nullable;
+
+@Name("NamespacedKey - Namespace/Key")
+@Description({"Gets the namespace or namespace key of a NamespacedKey object."})
+@Examples({"set {_items::*} to namespacedkey of every sword",
+		"loop {_items::*}:",
+		"\tbroadcast loop-value",
+		"\tbroadcast namespace of loop-value",
+		"\tbroadcast namespace key of loop-value"})
+@Since("INSERT VERSION")
+public class ExprNamespaceKeyOf extends SimplePropertyExpression<NamespacedKey, String> {
+
+	static {
+		register(ExprNamespaceKeyOf.class, String.class, "namespace [:key]", "namespacedkeys");
```

Comment:
I feel like this should be:
```suggestion
		register(ExprNamespaceKeyOf.class, String.class, "(namespace|:key)", "namespacedkeys");
```

Reply:
I didn't do that due to `key of %namespacedkeys%`, this was the same issue we had with lockable keys

Reply:
I don't know where that is used... that said, in my honest opinion, I don't think this expression is really needed.
Due to the fact that:
a) I can't see many people using it at all
b) you can easily stringily and split

Reply:
It's not used anywhere but it's just in the case of a conflict between anything with `key` being the only thing that's gonna be used for matching *aside from literals*